### PR TITLE
srtool hotfix specifying image

### PR DESCRIPTION
**Pull request by phpengine**
## Summary
Change report commands to specify a custom image, tagged as 

## Checklist
One line change to flag calling command

### CODE REVIEW

File: scripts/upgrade-runtime.sh [hunk 1]
```diff
@@ -101,10 +101,15 @@ if [[ $new_author_version != $((old_author_version + 1)) ]]; then
...
 
+echo "compiler prerequisites"
+docker pull paritytech/srtool:1.57.0
+docker tag $( docker image ls | grep 1.57.0 | awk '{ print $3 }' ) t3rn-srtool:1.60.0
+
 echo "🏭 compiling runtime wasm..."
 
 report="$( \
   srtool build \
+    --image t3rn-srtool:1.60.0 \
```
Comment (chiefbiiko) [inline scripts/upgrade-runtime.sh:112]:
```suggestion
    --image t3rn/srtool:1.60.0 \
```

File: scripts/upgrade-runtime.sh [hunk 2]
```diff
@@ -101,10 +101,15 @@ if [[ $new_author_version != $((old_author_version + 1)) ]]; then
   exit 1
 fi
 
+echo "compiler prerequisites"
```
Comment (chiefbiiko) [inline scripts/upgrade-runtime.sh:104]:
```suggestion
echo "🐳 monkey patching srtool-cli..."
```

**Review by chiefbiiko — CHANGES_REQUESTED** [review]
Some small cosmetics only

File: scripts/upgrade-runtime.sh [hunk 3]
```diff
@@ -101,10 +101,15 @@ if [[ $new_author_version != $((old_author_version + 1)) ]]; then
...
+docker pull paritytech/srtool:1.57.0
+docker tag $( docker image ls | grep 1.57.0 | awk '{ print $3 }' ) t3rn-srtool:1.60.0
```
[the remark this answers is not in the document]
Comment (phpengine) [reply scripts/upgrade-runtime.sh:106]: Correct on both points, adding and repushing these now

File: scripts/upgrade-runtime.sh (hunk 2 above)
Comment (phpengine) [reply scripts/upgrade-runtime.sh]: Added to commit

File: scripts/upgrade-runtime.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (phpengine) [reply scripts/upgrade-runtime.sh]: Added to commit

File: scripts/upgrade-runtime.sh (hunk 1 above)
Comment (phpengine) [reply scripts/upgrade-runtime.sh]: Added to commit

**Comment by chiefbiiko** [comment]
unfortunately, not the fix yet, retested upgrading but hitting same error => coming soon

**Pull request closed without merging**